src/i18n/strings/en_EN.json

```
@@ -790,6 +790,7 @@
     "Communities v2 prototypes. Requires compatible homeserver. Highly experimental - use with caution.": "Communities v2 prototypes. Requires compatible homeserver. Highly experimental - use with caution.",
     "New spinner design": "New spinner design",
     "Message Pinning": "Message Pinning",
+    "[WIP] Customizable keybindings": "[WIP] Customizable keybindings",
```

Very minor, however we do seem to favour british spelling in this file

```suggestion
    "[WIP] Customisable keybindings": "[WIP] Customisable keybindings",
```